packages/skeleton/src/lib/components/Accordion/AccordionItem.svelte

```diff
@@ -33,7 +32,9 @@
 	 * Provide a unique input id. Auto-generated by default
 	 * @type {string}
 	 */
-	export let id = String(Math.random());
+	export let id = '';
```

Comment:
@phamduylong instead of doing all these steps you implemented, we can achieve the same effect if you move the string `accordion-control-` to the default value.

so the only change would be:
```suggestion
	export let id = `accordion-control-${Math.random()}`;
```

Reply:
Dum dum me couldn't figure it out. Thanks, I'll implement this. Way cleaner.